### PR TITLE
Add v2 blog post

### LINKED ISSUE #719: Create an introduction to USWDS 2.0 (blog)
As we get ready to launch USWDS 2.0 we should understand what we want to communicate about its new features and functionality.

#### At the end of this issue, we should know the answers to the following
How do we communicate what's new in USWSD 2.0 on the home page for new visitors?
What do we need to do to more fully flesh out what's new in USWDS 2.0?
What new page or pages do we need to build?
What content do we need on this page or pages?

### PR DESCRIPTION
**Pull request by maya**
- Adds basic author field
- Did not figure out how to do feature image for social sharing though I see something in the meta include around it, not sure how it works / how to activate it.

[Preview - "News and updates" landing](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/add-v2-blog-post/about/updates/)
[Preview - Blog post](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/add-v2-blog-post/whats-new/updates/2019/04/08/introducing-uswds-2-0/)

Fixes #719.

### CODE REVIEW

File: _includes/post.html [hunk 1]
```diff
@@ -2,15 +2,18 @@
 <article id="{{ post.id | default: post.title | slugify }}"{% if include.excerpt %} class="post-excerpt padding-top-2 border-top-2px margin-top-4 measure-6"{% endif %}>
   <header class="post-meta">
     <p class="post-category site-subheading">
-      {{ post.category | default: post.tags[0] | default: layout.category }}
+      {{ post.tags[0] }}
```
Comment (maya) [inline _includes/post.html]: All of our categories were "About USWDS," the Updates page category so I default to the first tag and the categories seem to be back working.

**Comment by maya** [comment]
Were the blog landing page post titles also meant to have the larger heading styles?

**Comment by thisisdano** [comment]
I didn't want to go through the effort of decoupling them and figured it was ok.

File: _includes/post.html (hunk 1 above)
Comment (thisisdano) [reply _includes/post.html]: nice!

**Comment by thisisdano** [comment]
I'm gonna merge this branch into the homepage branch so we can release them at the same time.

**Pull request closed without merging**